Assets/MixedRealityToolkit.Tools/ReserializeAssetsUtility/ReserializeAssetsUtility.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public class ReserializeUtility
```

Comment:
Can you also add a doccomment about what this class is for (i.e. what it does, benefits, etc).

Usually we'd like to also have some docs on it as well, but in this case because it's pretty small maybe just having the doccomments would be sufficient.

Reply:
Oh whoops haha definitely

Reply:
Added